### DISCUSSION TITLE
New RELEASE  3.0

### DISCUSSION THREAD

**Discussion by felix2380** [helloworld1/FreeOTPPlus#256] [General]
I don't know if this is the right place but I'm posting here and I hope someone will see.I have the freeOTPplus app on my adroid phone and it's perfect,so far I haven't had any problems but I don't understand why the updates on GooglePlay are delayed with the new version 3.0.Does anyone know when version 3.0 will be released on android?I own version 2.9 from 2022 and we are at the end of 2023

**Comment by helloworld1** [comment]
Thanks for reporting. There is a release problem on Google play side. I have updated it there so hopefully Google play will have updated verison in a few days.

**Comment by felix2380** [comment]
Thank you very much for the answer!		On Fri, 03 Nov 2023 17:25:10 GMT, Howard ***@***.*** wrote:Thanks for reporting. There is a release problem on Google play side. I have updated it there so hopefully Google play will have updated verison in a few days.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>	
--3746333ace6b25aae980de6b2bf3dea16dc8e0a93494baf27bd0f6926416
Content-Transfer-Encoding: binary
Content-Type: text/html; charset=UTF-8

<p dir="auto" style="padding: 0px; margin: 0px; min-height: 1em;">Thank you very much for the answer!</p><p dir="auto" style="padding: 0px; margin: 0px; min-height: 1em;"></p><blockquote class="skiff_quote skiff_quote" data-skiff-sender="Howard Liberty" data-skiff-mail="true" isopen="false"><p dir="auto" style="padding: 0px; margin: 0px; min-height: 1em;">On Fri, 03 Nov 2023 17:25:10 GMT, Howard Liberty</p><p dir="auto" style="padding: 0px; margin: 0px; min-height: ***@***.*** wrote:</p><p dir="auto" style="padding: 0px; margin: 0px; min-height: 1em;"></p><p dir="auto" style="padding: 0px; margin: 0px; min-height: 1em;"></p><p dir="auto" style="padding: 0px; margin: 0px; min-height: 1em;">Thanks for reporting. There is a release problem on Google play side. I have updated it there so hopefully Google play will have updated verison in a few days.</p><p dir="auto" style="padding: 0px; margin: 0px; min-height: 1em;">—<br>Reply to this email directly, <a href
 ="https://github.com/helloworld1/FreeOTPPlus/discussions/256#discussioncomment-7469520" target="_blank" rel="noopener noreferrer nofollow" style="
    color: var(--text-link, rgb(239, 90, 60));
    background: var(--transparent-background);
    cursor: pointer;
    text-decoration: none;
">view it on GitHub</a>, or <a href="https://github.com/notifications/unsubscribe-auth/BDI67RTECM34GDNPTNIIYH3YCUSHFAVCNFSM6AAAAAA64WAVZGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TINRZGUZDA" target="_blank" rel="noopener noreferrer nofollow" style="
    color: var(--text-link, rgb(239, 90, 60));
    background: var(--transparent-background);
    cursor: pointer;
    text-decoration: none;
">unsubscribe</a>.<br>You are receiving this because you authored the thread.<img alt="" title="image" style="
    max-width: calc(100% - 15px);
    border: solid 1px transparent;
    margin: 0px 5px;
    display: inline-block;
" src="https://github.com/notifications/beacon/BDI67RQ34C74BYWE2MX2CKDYCUSHFA5CNFSM6AAAAAA64WAVZGWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAOH45A.gif" data-raw-src="https://github.com/notifications/beacon/BDI67RQ34C74BYWE2MX2CKDYCUSHFA5CNFSM6AAAAAA64WAVZGWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAOH45A.gif" height="1" width="1"><span style="color: transparent">Message ID: ***@***.***&gt;</span></p></blockquote>
--3746333ace6b25aae980de6b2bf3dea16dc8e0a93494baf27bd0f6926416--
